packages/stitch/tests/errors.test.ts

```diff
@@ -3,6 +3,7 @@ import { graphql, GraphQLError, buildSchema } from 'graphql';
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { stitchSchemas } from '@graphql-tools/stitch';
 import { assertSome, createGraphQLError, ExecutionResult, Executor } from '@graphql-tools/utils';
+import { ControlledError, makeRequest, startTestServer } from './utils';
```

Comment:
Do we need Apollo Server here? Can't we just use the schema itself?

Reply:
Maybe there is other way how to check the same stuff, but I am still not sure about the root cause of this issue. This is so far the minimal reproducible example I have. If I will have some spare time I can try to help debug the issue and once we know the root cause we will be able probably to provide simpler tests.